### PR TITLE
[14.0][MIG] event_mail: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT32127

Main changes:
- Adapt code to the use of computed writable

### CODE REVIEW

**Comment by ernestotejeda** [comment]
@sergio-teruel @chienandalu could you review, please?

File: event_mail/models/event.py [hunk 1]
```diff
@@ -16,28 +16,28 @@ def _default_event_mail_template_id(self):
...
     )
 
-    @api.onchange("event_mail_template_id")
-    def _onchange_event_mail_template_id(self):
-        vals = [(6, 0, [])]
-        if self.event_mail_template_id.exists():
-            for scheduler in self.event_mail_template_id.scheduler_template_ids:
-                vals.append(
+    @api.depends("event_mail_template_id")
+    def _compute_event_mail_ids(self):
```
Comment (ivantodorovich) [inline event_mail/models/event.py:20]: Nitpicking here but you could do this to split the water, it's slightly cleaner:

```py
records = self.filtered("event_mail_template_id")
without_template = self - records
```

**Review by ivantodorovich — APPROVED** [review]
LG

**Comment by ernestotejeda** [comment]
@ivantodorovich changes done

**Review by CarlosRoca13 — COMMENTED** [review]
LGTM

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**